tests/test_time_machine.py

```diff
@@ -412,6 +412,23 @@ def test_destination_datetime_tzinfo_zoneinfo_windows():
         assert time.timezone == orig_timezone
 
 
+def test_destination_datetime_tzinfo_pytz(monkeypatch):
+    try:
+        import pytz
+    except ImportError:
+        import subprocess
+
+        subprocess.run(["pip", "install", "pytz"])
```

Comment:
please don't run pip during tests, we can instead add `pytz` to `requirements.in` and recompile the test requirements